util/middleware.js

```diff
@@ -52,47 +60,3 @@ export async function secureFetchHandleErrors (url, accessToken, method = 'get',
     data
   }
 }
-
```

Comment:
I'm not a fan of this being moved to `components/LayoutWithAuth0.js`. It may not be very well organized here, but application logic shouldn't all be stuffed into one component. Maybe we either need to get better at implementing re-usable hooks.